env-tomap.go

```diff
@@ -1,5 +1,4 @@
-//go:build darwin || dragonfly || freebsd || linux || netbsd || openbsd || solaris
-// +build darwin dragonfly freebsd linux netbsd openbsd solaris
+//go:build !windows
```

Comment:
I think we still need the `// +build` due to older go versions

Reply:
since version 1.17, `go:build` is supported, so it makes no sense to support the tag for smaller versions - the minimum 1.17 is declared in go.mod

Reply:
ah, right!